### PR TITLE
Bindu patrick/score mechanism#52

### PR DESCRIPTION
**Pull request by vkanduku1995**
The score is displayed right after "Game Over" or "Congrats! You survived" text

### CODE REVIEW

File: Assets/Scripts/ScoreMechanism.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        //DontDestroyOnLoad(gameObject);
+        StartCoroutine(Test());
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        int score = PlayerPrefs.GetInt("score");
+        Debug.Log(score);
```
Comment (eldenpark) [inline Assets/Scripts/ScoreMechanism.cs:20]: How to see this printed out on console?

File: Assets/Scripts/GameManager.cs [hunk 2]
```diff
@@ -79,18 +81,21 @@ void Update()
                     CurrentGameState = GameState.Over;
                     StopCoroutine(operateTimer());
                     Time.timeScale = 0;
-                    statusText.text = "Congrats!\n You survived!";
+                    Uninfected = GameObject.FindGameObjectsWithTag("NORMAL_BALL");
+                    statusText.text = "Congrats!\n You survived! Score:"+ Uninfected.Length;
```
Comment (eldenpark) [inline Assets/Scripts/GameManager.cs:85]: ✅

File: Assets/Scripts/ScoreMechanism.cs (hunk 1 above)
Comment (vkanduku1995) [reply Assets/Scripts/ScoreMechanism.cs:20]: This script is not working as of now. I'm directly displaying the score on existing scene. This file was created to display the score on a new scene. Given the time constraint, I'm pushing it post midterm and will continue working on this file

Comment (eldenpark) [reply Assets/Scripts/ScoreMechanism.cs:20]: I see. I'd accept the PR as of now. Later in the meantime, we'd better include files that are relevant to the commit and only those that actually bring changes to the project. Thanks for the input!

**Review by eldenpark — APPROVED** [review]
👍

**Pull request merged**